pkg/lib/web/static_asset_resolver.go

```diff
@@ -102,6 +67,14 @@ func (r *StaticAssetResolver) StaticAssetURL(id string) (string, error) {
 	return staticAssetURL(r.Config.PublicOrigin, string(r.StaticAssetsPrefix), hashPath)
 }
 
+func (r *StaticAssetResolver) GeneratedStaticAssetURL(key string) (string, error) {
+	desc, _ := StaticAssetResources["generated-asset"]
```

Comment:
We can just refer the descriptor with `GeneratedAsset`, as we did in line 19. We can save a cast in line 73 too.